hoofball_1802_bronze.java

```diff
@@ -0,0 +1,71 @@
+/*
+Bill Zheng
+Time: 33 min
+USACO Test Cases: **********
+1-10 Difficulty: 5
+Reflection: This one was a rollercoaster because I actually spent over an hour initially trying to figure out if one method
+	worked and the code was really messy and ultimately the algorithm didn't work out in the end I think. So I had to come up
+	with a different method after spending so much time on the first solution and I ended up with the code below. It's still
+	very messy but I'm just happy that I'm done with this problem. It's very deceiving because it looks easy but isn't.
+ */

```

Comment:
Try describe your algorithms concisely. Practice how to describe your algorithm effectively until you believe a five-year old can understand. :)

Reply:
I think for my first algorithm I could describe it easily but there were always edge cases where it wouldn't work properly and so it got super complex and never worked. I think my new final algorithm is much easier to describe and understand.

Reply:
> there were always edge cases

Then it means your description has holes, and thus, incorrect